dali/common.h

```diff
@@ -74,7 +74,7 @@ enum DALIInterpType {
 /**
  * @brief Supported image formats
  */
-enum DALIImageType { DALI_RGB = 0, DALI_BGR = 1, DALI_GRAY = 2 };
+enum DALIImageType { DALI_RGB = 0, DALI_BGR = 1, DALI_GRAY = 2, DALI_YCBCR = 3 };
```

Comment:
```suggestion
enum DALIImageType { DALI_RGB = 0, DALI_BGR = 1, DALI_GRAY = 2, DALI_YCbCr = 3 };
```

Reply:
I thought about it first, but all our enumerators are in upper case